tests/storage/restore.go

```diff
@@ -740,6 +740,18 @@ var _ = SIGDescribe("[Serial]VirtualMachineRestore Tests", func() {
 					snapshotStorageClass,
 				))
 
+				Eventually(func() error {
+					return libnet.WithIPv6(console.LoginToCirros)(vmi)
```

Comment:
do you really need WithIPv6 here ?
I am not sure, LoginToCirros should be enough

Reply:
I did what is done the login in all the other tests

Reply:
You don't need to config here IPv6, the fact its on the other tests, is either that they need it, or they did copy paste as well

Reply:
I updated the PR name. From what I see the login was previously without the ipv6 and it was changed not only in restore tests but in a lot of tests to use it. If it works ok I would rather leave it as is.

Reply:
Thanks